### ISSUE TITLE
Missing input/output parameter edition panel

### ISSUE THREAD

**Issue by szerie** [camunda/camunda-modeler#2018]
__Describe the Bug__

If template that is used contains configuration like:

```
    "entriesVisible": {
      "_all": false,
      "inputs": true,
      "outputs": true
    }
```
then after selecting input or output parameter additional panel containing its name, type and value is not visible:

![MissingPanel1](https://user-images.githubusercontent.com/57395064/99076394-5b8cd380-25bb-11eb-8396-b03b0bafaab2.PNG)

If _all flag is set to true then panel is visible:
![MissingPanel2](https://user-images.githubusercontent.com/57395064/99076463-75c6b180-25bb-11eb-9f35-5fdf5252f61c.PNG)

Maybe some flag is missing in template visibility configuration, but unfortunately I did not found anything in Camunda Modeler documentation.

__Steps to Reproduce__

1. Create template containing visibility configuration like:
```
    "entriesVisible": {
      "_all": false,
      "inputs": true,
      "outputs": true
    }
```
2. Open Camunda Modeler.
3. Apply newly created template on any task.
3. Select mentioned task.
4. Go to input/output parameter tab.
5. If any parameter already exists (applied from template) then select it. If not then create new parameter using "+" button and select it.
6. Parameter properties panel is not visible. It is not possible to set parameter properties.

__Expected Behavior__

Parameter properties panel is visible and enabled after selecting any input or output parameter.

__Environment__

- OS: Windows 10
- Camunda Modeler Version: 4.2.0

**Comment by philippfromme** [comment]
We've got two issues here.

1. The _Input/Output_ tab will not be rendered if an element template was applied due to this change: https://github.com/bpmn-io/bpmn-js-properties-panel/pull/372/commits/c1b965b20d2dc1b554e7d9ca086a9da850913219 As a result of this change the tab will not contain any entries and therefore cannot be made visible through `entriesVisible`.

2. The IDs of the IO entries have changed so `appliesTo: { inputs: true; outputs: true; }` doesn't work anymore. In fact, there are now multiple entries for both inputs and outputs with some of the IDs being generated dynamically which breaks the entire concept of `entriesVisible`.

![image](https://user-images.githubusercontent.com/7633572/99416488-2ead2900-28f9-11eb-8d4c-a6098af89eeb.png)

@camunda/modeling-dev What are we going to do about this?

**Comment by tamaserdfalvi** [comment]
As a sidenote a list of possible entries would be nice too.

**Comment by nikku** [comment]
> As a sidenote a list of possible entries would be nice too.

We intentionally do not maintain such list. We won't do that in the future, too. Using `entriesVisible` is an expert feature and hooks into certain internals of how the properties panel works (i.e. IDs of sections that may change over time). Be aware of that risk, as you use this feature.

**Comment by tamaserdfalvi** [comment]
Fair enough, now I don't even see it referenced in the documentation. Might have been there before, though, otherwise I think I wouldn't have known about that too :)

**Comment by nikku** [comment]
As discussed internally we'll move this issue to our backlog. We need to gather further feedback before we decide what an appropriate solution can look like.
